data/json/items/comestibles/raw_veggy.json

```diff
@@ -209,24 +209,61 @@
   {
     "type": "COMESTIBLE",
     "id": "corn",
-    "name": { "str_sp": "corn" },
-    "weight": "153 g",
+    "name": { "str": "corn cob" },
+    "weight": "690 g",
     "color": "light_green",
     "spoils_in": "5 days",
     "comestible_type": "FOOD",
     "symbol": "%",
     "healthy": 1,
     "calories": 132,
-    "description": "Delicious golden kernels.",
+    "description": "A corn cob full of delicious golden kernels.  You can eat them on the cob or shell it for cooking.",
     "price": 170,
     "//": "Thinking this is a single ear here.",
     "price_postapoc": 50,
     "material": [ "veggy" ],
     "volume": "750 ml",
+    "flags": [ "EATEN_HOT", "RAW" ],
+    "vitamins": [ [ "vitA", 2 ], [ "vitC", 12 ], [ "iron", 4 ] ],
+    "use_action": { "type": "transform", "target": "empty_corn_cob", "moves": 500 }
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "corn_kernels",
+    "name": { "str_sp": "corn kernels" },
+    "weight": "210 g",
+    "color": "yellow",
+    "spoils_in": "5 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 1,
+    "calories": 132,
+    "description": "Delicious golden kernels.",
+    "price": 200,
+    "price_postapoc": 60,
+    "material": [ "veggy" ],
+    "volume": "285 ml",
     "flags": [ "EATEN_HOT", "SMOKABLE", "RAW" ],
-    "smoking_result": "dry_veggy",
+    "smoking_result": "dry_corn",
     "vitamins": [ [ "vitA", 2 ], [ "vitC", 12 ], [ "iron", 4 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "empty_corn_cob",
+    "name": { "str_sp": "empty corn cob" },
+    "weight": "480 g",
+    "color": "yellow",
+    "spoils_in": "5 days",
+    "comestible_type": "FOOD",
```

Comment:
Shouldn't it have calories? I would imagine they are unpleasant to eat when fully ripe, don't remember if I tried it.

Reply:
They are pure cellulose, and cannot be digested by humans. I'm actually going to need to move it ouy of comestibles after.

Reply:
Comestibles _may_ need calories value but I'm not sure. Sounds good. Do you mean later in this PR or do you want to postpone it until a later PR?

Reply:
I will do it now, I just don't know exactly where to put it... probably on the other\other.json? and finally the fish bait won't be alone in there.

Reply:
Nevermind, someone convinced me to move it back to comestibles, but I'm adding a morale penalty. 

Also it can't have comestible traits such as "spoils_in" if it's not on comestibles, apparently